src/core/context/node.js

```diff
@@ -209,23 +212,23 @@ export class ContextNode {
     /**
      * See `parent` description.
      *
-     * @package {?Array<!ContextNode>}
+     * @package {?ContextNode[]}
      */
     this.children = null;
 
-    /** @package {?Map<string, !GroupDef>} */
+    /** @package {?{[key: string]: GroupDef}} */
     this.groups = null;
 
     /** @package {!Values} */
     this.values = new Values(this);
 
-    /** @private {?Map<!SID, !./subscriber.Subscriber>} */
+    /** @private {?{[key: SID]: ./subscriber.Subscriber}} */
     this.subscribers_ = null;
 
     /** @private {boolean} */
     this.parentOverridden_ = false;
 
-    /** @const @private {function()} */
```

Comment:
Can you update all of these lines like this to instead be:
```js
/**
 * @type {function():void}
 * @const
 * @private
 */
```

The alternative is to make it `any`, which is what this is currently doing.

Reply:
Done

Reply:
Thank you!